app/models/user.rb

```diff
@@ -0,0 +1,45 @@
+class User < ApplicationRecord
+
+  attr_accessor :remember_token
+  before_save { self.email = email.downcase }
+  validates :name,  presence: true, length: { maximum: 50 }
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  validates :email, presence: true, length: { maximum: 255 }, format: { with: VALID_EMAIL_REGEX }, uniqueness: { case_sensitive: false }
```

Comment:
đưa 255 vào setting và thừa dâu cách sau {}.
A check toàn bộ file nha.

Reply:
đã xong